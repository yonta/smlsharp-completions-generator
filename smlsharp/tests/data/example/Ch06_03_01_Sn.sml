val n = 10;
val Sn = (n * (n + 1) * (2*n + 1)) div 6;
