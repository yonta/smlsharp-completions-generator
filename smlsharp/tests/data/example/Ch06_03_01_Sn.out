val n = 10 : int

val Sn = 385 : int
